sdk/textanalytics/tests.yml

```diff
@@ -9,5 +9,6 @@ jobs:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        AZURE_TEXT_ANALYTICS_ENDPOINT: $(java-textanalytics-sdk-endpoint)
+        AZURE_TEXT_ANALYTICS_ENDPOINT: $(java-azure-textanalytics-sdk-endpoint)
+        AZURE_TEXT_ANALYTICS_SUBSCRIPTION_KEY: $(java-textanalytics-test-subscription-key)
```

Comment:
Should this value be output by the resource generator?

Reply:
These values should not exist here anymore as they would be output variables from the test-resources.json file itself.